src/utils/components/PendingChanges/hooks/usePendingChanges.tsx

```diff
@@ -152,7 +152,7 @@ export const usePendingChanges = (
           <CPUMemoryModal isOpen={isOpen} onClose={onClose} onSubmit={onSubmit} vm={vm} />
         )),
       ),
-      hasPendingChange: !isInstanceTypeVM(vm) && cpuMemoryChanged && restartRequired(vm),
+      hasPendingChange: !isExpandableSpecVM(vm) && cpuMemoryChanged && restartRequired(vm),
```

Comment:
user can override preferences - I believe we need a strict `isInstanceTypeVM` check here